source/uwp/SharedRenderer/lib/XamlHelpers.cpp

```diff
@@ -242,11 +243,25 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering::XamlHelpers
         return content;
     }
 
-    winrt::Image CreateBackgroundImage(winrt::AdaptiveRenderContext const& renderContext, winrt::hstring const& url)
+    winrt::Image CreateBackgroundImage(winrt::AdaptiveRenderContext const& renderContext, winrt::hstring const& url, winrt::TileControl const& tileControl)
     {
         try
         {
             auto imageUrl = GetUrlFromString(renderContext.HostConfig(), url);
+            auto imagePath = HStringToUTF8(imageUrl.Path());
+            bool isImageSvg = imagePath.find("svg") != std::string::npos;
+
+            if (isImageSvg)
```

Comment:
this if statement and one following is functionally correct, but incorrect semantically.
could you change it to if (isImageSve) {} else {//mening we are dealing with bitmap; if (data) {} else {}}?